source/ways-of-working/new-component/index.html.md.erb

```diff
@@ -19,5 +19,5 @@ weight: 3
 - [Flux config](gitops-flux.html#application-config-in-flux)
 - [Production approvals](https://github.com/hmcts/cnp-jenkins-config/blob/master/environment-approvals.yml)
 - [Path-to-live](../path-to-live/)
-- [Accessing apps via the VPN](ssh-via-vpn.html)
+- [Accessing apps via the VPN](connect-via-vpn.html)
```

Comment:
This doesn't appear to be in the right section, very few applications will need this.

Maybe create a new one `Guides` ?